.circleci/config.yml

```diff
@@ -444,6 +444,19 @@ jobs:
           command: |
             set -xu
             yum -y install java-1.8.0-openjdk
+      - run:
+          name: "Install testbench (GCS dependency)"
+          command: |
+            set -xu
+            ./scripts/install_gcs_testbench.sh
+      - run:
+          name: Build
+          command: |
+            mkdir -p .ccache
+            export CCACHE_DIR=$(realpath .ccache)
+            ccache -sz -M 5Gi
+            source /opt/rh/gcc-toolset-9/enable
+            make release EXTRA_CMAKE_FLAGS="-DVELOX_ENABLE_ARROW=ON -DVELOX_ENABLE_PARQUET=ON -DVELOX_ENABLE_HDFS=ON -DVELOX_ENABLE_S3=ON -DVELOX_ENABLE_GCS=ON" AWSSDK_ROOT_DIR=~/adapter-deps/install GCSSDK_ROOT_DIR=~/adapter-deps/install NUM_THREADS=16 MAX_HIGH_MEM_JOBS=8 MAX_LINK_JOBS=8
```

Comment:
DVELOX_ENABLE_S3 should not be needed for GCS.

Reply:
Why a separate `Build` step? Can add to the `Build including all Benchmarks` below?

Reply:
Can you also make a note of the increase in build and test times with this change in the description?